File: Go/uri_1043.go
package main

import "fmt"

func main() {
	var v [3] float64
	for i := 0; i < len(v); i++ {
		fmt.Scan(&v[i])
	}
	if v[0]+v[1] > v[2] && v[0]+v[2] > v[1] && v[1]+v[2] > v[0] {
		fmt.Printf("Perimetro = %.1f\n", v[0]+v[1]+v[2])
	} else {
		fmt.Printf("Area = %.1f\n", (v[0]+v[1])*v[2]/2.0)
	}

}

File: Python/uri_1043.py
a, b, c = map(float, input().split(' '))
if a + b > c and a + c > b and b + c > a:
    p = a + b + c
    print('Perimetro = %0.1f' % p)
else:
    s = ((a + b) * c) / 2.0
    print('Area = %0.1f' % s)
